### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by PrabhanjanPM**
1. Updated driver to access the attacks
2. Problem in importing Pollard-Rho attack

### CODE REVIEW

**Comment by chinmaydd** [comment]
@PrabhanjanPM Good. Have you tested it locally?

File: daedalus/attacks/wieners.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (kishorbhat) [inline daedalus/attacks/wieners.py]: Random newline.

File: daedalus/daedalus.py [hunk 2]
```diff
@@ -28,15 +28,34 @@ def loadprivkey(self, args, type=None):
...
-
 	def attack(self, code):
-		# For example purposes only! Remove later.
-		if code == 'ex':
-			# Call the appropriate attack module
-			pass
+		if code == 'wieners':
+			out=attacks.wieners.attack({'n':self.n,'e':self.e})
+			print "Results" ,
+			print out['results']
```
Comment (kishorbhat) [inline daedalus/daedalus.py:37]: Could be written as `print "Results", out['results']`

File: daedalus/daedalus.py [hunk 3]
```diff
@@ -28,15 +28,34 @@ def loadprivkey(self, args, type=None):
...
+			print "Errors" ,
+			print out['errors']
```
Comment (kishorbhat) [inline daedalus/daedalus.py:39]: Similar to previous comment.

**Comment by PrabhanjanPM** [comment]
I don't have sage so I couldn't test . But I think my attack should work
fine.

If I change a script from .sage to .py, do I need to add any extra
statements?

File: daedalus/daedalus.py [hunk 4]
```diff
@@ -28,15 +28,34 @@ def loadprivkey(self, args, type=None):
...
+			#Do Something
+
+		elif code=='pollard_rho':
+			out=attacks.pollard_rho_algorithm.attack({'n':self.n,'e':self.e})
+			print "Results" ,
+			print out['results']
+			print "Errors" ,
+			print out['errors']
```
Comment (kishorbhat) [inline daedalus/daedalus.py:53]: Previous comment applies to all of these as well.

File: daedalus/daedalus.py [hunk 5]
```diff
@@ -59,15 +77,14 @@ def shell():
 			'''
 		elif entered == 'help attacks':
 			print '''
-Attack                       Code
-------                       ----
-Example                      ex
+Attack       lp                Code
```
Comment (kishorbhat) [inline daedalus/daedalus.py]: What's "lp"?

**Comment by kishorbhat** [comment]
> I don't have sage so I couldn't test . But I think my attack should work
> fine.

That's really not how we should be testing.

Please address the inline comments.

**Comment by chinmaydd** [comment]
@PrabhanjanPM After you are done with the fixes as suggested by @therealkbhat , please test your code by choosing appropriate public and private key and accessing your code from daedalus.py file.

**Pull request merged**